packages/react-ui/src/index.d.ts

```diff
@@ -34,6 +34,7 @@ import AppBar from './components/organisms/AppBar/AppBar';
 import LabelWithIndicator from './components/atoms/LabelWithIndicator';
 import { getCartoColorStylePropsForItem } from './utils/palette';
 import Avatar from './components/molecules/Avatar';
+import { ICON_SIZE_S, ICON_SIZE, ICON_SIZE_M } from './theme/themeconstants';
```

Comment:
I have minor doubt about naming of those constants.

Looks like`ICON_SIZE_S` means small, and `ICON_SIZE_M` is medium ... then `ICON_SIZE` is normal? So, normal is smaller than medium ?

Maybe we could find more explicit and understantable names for these constants?

Reply:
Done! I've changed it to be more explicit